cryptofeed/rest/gdax.py

```diff
@@ -174,8 +191,16 @@ def _delete_order(self, order_id=None):
         if order_id is not None:
             endpoint = '{}/{}'.format(endpoint, order_id)
 
-        header = self._generate_signature(endpoint, "DELETE", body=json.dumps(body))
+        header = self._generate_signature(endpoint, "DELETE")
```

Comment:
you're creating a var `header` but below using `headers` in the method call. I think this is a bug - guessing this one isn't testable on the sandbox?